src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -143,6 +144,33 @@ public void checkDuplicateCqlLibraryName(String cqlLibraryName) {
     }
   }
 
+  public void validateMeasurementPeriod(Date measurementPeriodStart, Date measurementPeriodEnd) {
+    try {
+      SimpleDateFormat df = new SimpleDateFormat("MM-dd-yyyy");
+      df.format(measurementPeriodStart);
+      df.format(measurementPeriodEnd);
```

Comment:
Seems to be unused code?

Reply:
This code basically validates if measurement periods are null, also validates month and date(i.e., to check if months>12 or date>31). If line 150 or 151 is not executed it throws an exception